README.adoc

```diff
@@ -307,6 +307,31 @@ _build/features_report.html_. They can be opened via:
 $ bundle exec rake features:open_report
 ---------------------------------------
 
+=== Asciidoc
+
+The documentation of this plugin is written in http://asciidoctor.org[Asciidoc]. If you need some syntax help
```

Comment:
@hferentschik Forgot `to` in second sentance.

"If you need some syntax help `to` start with ...."